isis/src/clementine/apps/clemhirescal/clemhirescal.xml

```diff
@@ -9,14 +9,17 @@
       camera. It does so through the use of some constants based on offset and
```

Comment:
@AustinSanders Perhaps add some details to the description describing the (very small) differences in output radiances that result from switching from the look up tables to the modeled function?
This would address questions from @scsides  in his review.

The equation for the A filter yields identical k values that are identical to the ones in Table 5 from Robinson et al. (2003), therefore the pixel values in the output image are the same as the would be running the earlier code.

The equation for the D filter yields k values that differ from the hard-coded values by less than 1 standard deviation as reported in Table 6 of Robinson et al. (2003). The radiance values that result from computed k values differ by no more than 0.581% (that is, less than 1 percent) than radiance values computed from the hard-coded k values.

Reply:
@dpmayerUSGS Can you ensure that I properly captured the intended changes?

Reply:
@AustinSanders The changes to the documentation look good to me!